src/components/Sidebar.tsx

```diff
@@ -108,6 +109,12 @@ const sidebarItems: SidebarItem[] = [
     regex: /^\/groups$/,
     roles: 'GroupManager',
   },
+  {
+    name: '人才地图',
+    path: '/map',
+    icon: MdMap,
+    regex: /^\/map/,
+  }
```

Comment:
let's not add this menu item until the full map feature is complete. otherwise users on production will see an unfinished feature. feel free to add the menu item to non-production environment though (local and staging).